client/src/modules/payment/store/getters.ts

```diff
@@ -10,6 +10,9 @@ export default {
   [types.PAYMENT_COMPLETE_MODAL_IS_VISIBLE]: state => state[STATE_KEY].isPaymentCompleteVisible,
   [types.PAYMENT_IS_IN_PROGRESS]: state => state[STATE_KEY].isProcessing,
   [types.GET_SBC_PAYMENT]: state => state[STATE_KEY].sbcPayment,
+  [types.GET_SBC_PAYMENT_STATUS]: state => (state[STATE_KEY].sbcPayment)
```

Comment:
"types" should be capitalized to distinguish it from a property/variable.

Reply:
Not enforced by eslint, and those are not actually 'types' referenced in those files, they're exported string constants so technically they are just exported variables. I could have written 'action_dictionary.<type>' or 'all_actions.<type> I guess but yeah...